### ISSUE TITLE
QRange marker-labels property has some type error

### ISSUE THREAD

**Issue by jeremyjone** [quasarframework/quasar#12164]
### What happened?

I'm using the `marker-labels` property of `q-range` with `TypeScript`, but I always have a type error.

![image](https://user-images.githubusercontent.com/37676231/150745096-b9efbcca-b1d3-48ab-9065-4a5d9e039551.png)

I looked at the type source code, they look like they're the same.

![image](https://user-images.githubusercontent.com/37676231/150745524-12a05c54-0a0e-4ce7-88c9-260ab6cff152.png)

I found that this statement is the same as `q-slider`, and I tried it and found that the same error would be reported.

To fix the problem, I upgraded to the latest version(`v2.4.12`), but it didn't help.

I think this should be a bug.

### What did you expect to happen?

I just want to eliminate the bug.

### Reproduction URL

I created a minimal instance on my [GitHub](https://github.com/jeremyjone/quasar-test)

### How to reproduce?

1. Open project and locate to `src/components/HelloWorld.vue` file with vscode
2. use `marker-labels` property (I used the example parameters provided in the official documentation)
3. there has a error line

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

TypeScript Support

### Platforms/Browsers

Other

### Quasar info output

```shell
No. Everything looks OK. But only has a type error, can not build.
```

**Comment by MilosPaunovic** [comment]
Maybe @yusufkandemir can take a look at this when he gets a chance.

**Comment by rstoenescu** [comment]
Taking care of it.

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.4.13.
